powermon/__init__.py

```diff
@@ -125,13 +131,13 @@ def main():
         return None
 
     # Build configuration from defaults, config file and command line overrides
```

Comment:
do we even need all this? maybe no sample_config, build objects from config file and change if overridden

Reply:
That makes sense.